src/main/java/com/github/rakhmedovrs/server/SocketConnectionHandler.java

```diff
@@ -41,59 +46,64 @@ public void run() {
 				 PrintWriter output = new PrintWriter(new OutputStreamWriter(outputStream, StandardCharsets.UTF_8), true)) {
 				Integer randomNumber = random.nextInt(RANDOM_RANGE + 1); //upper bound is excluded
 				logIfNeeded("Server decided to generate {} as a number to be guessed", randomNumber);
-				output.println("Server generated random number in range [0," + RANDOM_RANGE + "]. Guess it");
+				output.println(generateServerGuessResponseAsString(GuessResult.UNDEFINED));
```

Comment:
perhaps if u guessed some number , it is would be more convenient to reply with its range, not UNDEFINED.

Reply:
Fixed